spec/rails/doc/smart/openapi.yaml

```diff
@@ -172,6 +190,7 @@ paths:
                 id: 1
                 name: access
                 description: logs
+                no_such_field_response: true
```

Comment:
Note) This is added but does not appear in the`expected.json` because it is removed because of the pattern `6) paths.*.*.responses.*.content.application/json.example.*`